docs/get-started/getting-started-databricks.md

```diff
@@ -92,15 +96,17 @@ larger dataset if needed. You can find the links to the datasets at
 
 ```bash
 %sh
+
+USER_ID=<your_user_id>
```

Comment:
```suggestion
```

Reply:
@jlowe I am fine changing `${USER_ID}` to `${USER}`, but curious why we should remove `USER=<your_user_id>` from the top of the script?

Reply:
Because `USER` is a standard enviornment variable, it does not need to be set.

Reply:
I ran in the Databricks notebook without setting $USER_ID and instead using the $USER variable.  Unfortunately in that environment the notebook is run as `root`.  In a shared cluster environment all users would be writing to the same /Users/root folder.  

```bash
%sh

echo $USER
```
`root`

Reply:
ah nevermind then, keep it as is with the separate `USER_ID` variable.  We shouldn't smash `USER` with a new value.